utils.ts

```diff
@@ -2,37 +2,37 @@ import "allocator/tlsf";
 import { allocateUnsafe, copyUnsafe, HEADER_SIZE } from "internal/string"
 import { ErrorCode, debug } from "./index"
 
-
+@inline
```

Comment:
What does "@inline" do exactly?

Reply:
It force function inlining. The same as `#[inline]` in Rust of `__attribute__((always_inline))` in GCC. You can read about [more](https://en.wikipedia.org/wiki/Inline_function)